src/style.css

```diff
@@ -1 +1,75 @@
-/* Styles go here */
+body {
+  font-family: "Roboto", sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  box-sizing: border-box;
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+  padding: 10px 50px;
```

Comment:
You don't need to apply vertical padding here. You could just align the logo to the center

Reply:
didn't fix it